### PR TITLE
EDD-40: Adding github page

### PR DESCRIPTION
**Pull request by trevorlang**
# Overview

### What is the feature?

Moves the download page from Earthdata Search to a github page.

### What is the Solution?

The Earthdata Search download page was recreated using a vanilla Vite starter. It's mainly comprised of html, scss, and a small browser detection library. Some simple linting is in place as well. Github actions are in place to build and deploy to Github Pages. Values for the download links and sizes for the various OS's have been added as Vite environment variables in the `.env` file.

### What areas of the application does this impact?

Everything will be contained on the `gh-pages` branch.

# Testing

### Reproduction steps

- **Environment for testing:** N/A
- **Collection to test with:** N/A

1. Visit the Github page
2. Confirm the primary download link matches your OS
3. Confirm the links are the latest download links

### Attachments

Please include relevant screenshots or files that would be helpful in reviewing and verifying this change.

# Checklist

- [ ] I have added automated tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have bumped the `version` field in package.json and ran `npm install`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+1. Run `npm install`
+2. Run `npm run dev`
+
+## Environment Variables
+
+Environment variables are used to define the download location and download size for each of the images. These values are defined in the `.env` file.
+
+## Icons
+
+In order to keep the page as lean as possible, SVG icons have been imported from Font Awesome. The icon SVG XML was pulled directly from the [Font Awesome](https://fontawesome.com/icons/l) website and placed in the `/public/images/icon/` directory.
```
Comment (trevorlang) [inline README.md:16]: Not relevant anymore

File: js/util/setOSClassName.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import { UAParser } from 'ua-parser-js'
+
+export function setOSClassName(element) {
+  const parser = new UAParser(window.navigator.userAgent)
+  const { name: osName } = parser.getOS()
+  // const osName = 'Windows'
```
Comment (trevorlang) [inline js/util/setOSClassName.js:6]: Remove

File: index.html [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+      </a>
+      <div class="primary-download__meta">
+        <span>
+          %VITE_OS_MACOS_DMG_DOWNLOAD_SIZE% (.dmg)
+        </span>
+        <em class="primary-download__caveat"> for Intel-based Macs</em>
+      </div>
+    </section>
+    <section class="primary-download primary-download--windows">
+      <a class="button primary-download__link" href="%VITE_OS_WINDOWS_EXE_DOWNLOAD_LINK%">
```
Comment (trevorlang) [inline index.html:37]: Add the class needed to make this a primary button

File: index.html [hunk 4]
```diff
@@ -0,0 +1,231 @@
...
+        Download for Windows
+      </a>
+      <div class="primary-download__meta">
+        <span>
+          %VITE_OS_WINDOWS_EXE_DOWNLOAD_SIZE% (.exe)
+        </span>
+      </div>
+    </section>
+    <section class="primary-download primary-download--linux">
+      <a class="button primary-download__link" href="%VITE_OS_LINUX_APPIMAGE_DOWNLOAD_LINK%">
```
Comment (trevorlang) [inline index.html:54]: Fix this one too

**Review by macrouch — APPROVED** [review]

**Review by dpesall — APPROVED** [review]

**Pull request merged**